ofl/grandstander/METADATA.pb

```diff
@@ -0,0 +1,32 @@
+name: "Grandstander"
+designer: "Tyler Finck"
+license: "OFL"
+category: "SANS_SERIF"
```

Comment:
Font looks more like a display font than a sans. If you agree, change the category to "DISPLAY"

Reply:
ok